Lib/opcode.py

```diff
@@ -247,6 +247,7 @@ def jabs_op(name, op):
     "STORE_ATTR_SLOT",
     "STORE_ATTR_WITH_HINT",
     # Super instructions
+    "UNPACK_SEQUENCE_ST",
```

Comment:
Please keep with the naming convention and rename this to ``UNPACK_SEQUENCE__STORE_FAST``. Thank you.

Reply:
Done.